### PR TITLE
Add new Plasma frigate on map

### PR DESCRIPTION
**Pull request by Ultima1990**
New Plasma Frigate sprites created by Croak.

Map changes:
Removed old Plasma Frigate dungeon.
Extended leftmost Castelia City dock.

Here's what'll happen:
On clearing Virbank Gym the Plasma Frigate appears at the Castelia dock. No function, just looks pretty, you do get a li'l message if you click it.
On clearing Driftveil Gym the Plasma Frigate will disappear from the Castelia dock and move to Driftveil City. Once again no function except there's a message if you click it.
On clearing Opelucid Gym the Plasma Frigate will disappear from Driftveil City and appear in airship mode over Opelucid City. Clicking it will access the Team Plasma Assault dungeon. The dungeon can also still be accessed from inside Opelucid City.
On clearing the Team Plasma Assault dungeon the Plasma Frigate airship will disappear from Opelucid City and appear in normal boat mode next to Seaside Cave. It will be locked until Humilau gym is defeated and route 22 is cleared as usual. Clicking it will access the Plasma Frigate dungeon.
On clearing the Plasma Frigate dungeon the Plasma Frigate will disappear from Seaside Cave and appear in airship mode over Giant Chasm. It will be an extension of the Giant Chasm dungeon. Just nice for flavor.
On clearing Giant Chasm the Plasma Frigate airship will disappear from Giant Chasm. The questline is concluded. The Plasma Frigate next to Seaside Cave and the Plasma Frigate airship over Opelucid City will reappear for replaying and achievement hunting. Not strictly necessary for the Opelucid one because the Team Plasma Assault is also accessible from inside Opelucid City, but still.

### CODE REVIEW

File: src/components/UnovaSVG.html [hunk 1]
```diff
@@ -846,4 +831,87 @@
         })
     %>
 
+    <!--Plasma Frigate 1, Castelia City-->
+    <%- include('templates/mapOverlay',
+        { name: "A very normal frigate"
+        , x: 39, y: 52
+        , width: 4, height: 9
+        , databind: `attr: { class: App.game.statistics.gymsDefeated[GameConstants.getGymIndex('Virbank City')]() >= 1 },
```
Comment (Jaaslet) [inline src/components/UnovaSVG.html:839]: This class-attribute does not do anything, so let's remove it

**Review by Jaaslet — CHANGES_REQUESTED** [review]
Looks good. I have two ideas and a small code-change.
- The MoveToDungeon in Opelucid City spoils the surprise
- Shall we remove the plastma-boss from giant chasm, after you defeated it first time? For flavor. We can just set a reverse dungeoncleared requirement and hide = true. Should work.

File: src/components/UnovaSVG.html (hunk 1 above)
Comment (Ultima1990) [reply src/components/UnovaSVG.html]: Removed class-attribute, removed Opelucid City MoveToDungeon.
Not gonna remove Ghetsis from Giant Chasm after first clear. We don't remove story bosses from other dungeons.

**Review by Jaaslet — APPROVED** [review]
Look good. Thanks for the PR

**Pull request merged**